### PR TITLE
WIP [13.0][MIG] base_user_locale

### PR DESCRIPTION
**Pull request by Tonow-c2c**
WIP :hammer: 

- [x] check what should I do change to tackle this comment
- [x] check https://github.com/OCA/server-ux/pull/158/files#diff-4b4975f36e2ac07b8f87ba79597fd23bR21-R23 about https://github.com/odoo/odoo/pull/36532
- [ ] test it

### CODE REVIEW

File: base_user_locale/README.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+.. contents::
+   :local:
+
+Known issues / Roadmap
+======================
+
+In v13 it is expected that this module will need significant rework.
+Until `#36532 <https://github.com/odoo/odoo/pull/36532>`_ is merged,
+fix is included into this module as well.
```
Comment (ivantodorovich) [inline base_user_locale/README.rst]: What about this?

is this significant rework still required?
Apparently that pr was merged. 

If that's not longer a known issue, please remove it from the readme

File: base_user_locale/static/src/js/session.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+         */
+        load_modules: function() {
+            var self = this;
+            return this._super.apply(this, arguments).then(function() {
+                return self.configure_locale();
+            });
+        },
+        /**
+         * See https://github.com/odoo/odoo/pull/36532
+         */
```
Comment (ivantodorovich) [inline base_user_locale/static/src/js/session.js]: Maybe that's what the known issue was mentioning?
Is it still required?

**Review by ivantodorovich — COMMENTED** [review]
Thanks for the work @Tonow-c2c 

It looks like pre-commit is not initilized in your local clone, that's why travis is failing:
https://odoo-community.org/blog/the-oca-blog-1/post/black-isort-pre-commit-97

Also I left a few comments

**Comment by Tonow-c2c** [comment]
thanks @ivantodorovich  for your review, sorry I didn't mention it still a wip PR I will check your comments sooner 
I add in the description a TODO list of things to check for this module migration, be my guest to improve it :wink:

**Comment by MiquelRForgeFlow** [comment]
Superseded by https://github.com/OCA/server-ux/pull/337.

**Pull request closed without merging**